cloudvolume/cloudvolume.py

```diff
@@ -211,15 +215,15 @@ def commitInfo(self):
     return self.commit_info()
 
   def commit_info(self):
-    if self._protocol == 'boss':
+    if self.path.protocol == 'boss':
```

Comment:
All this line is some sort of silent failure ,right?
info file cannot be changed for boss, but when we are trying to save the change we do nothing
the user thinks she actually commited the changes.

This applies to refresh_provenance and others places

Reply:
When I originally wrote this, it was the fastest way to get `boss` working without needing to rewrite a bunch of tasks that call `commit_info`. The right way to do it is probably to actually try committing the info file to BOSS. BOSS also doesn't support provenance files per se. I think we're going to address this in a refactoring that separates cutout services from Precomputed based key-value stores.